nix/packages/make-dhcpd/package.nix

```diff
@@ -0,0 +1,13 @@
+{
+  writeShellApplication,
+  dnsmasq,
+}:
+writeShellApplication {
```

Comment:
The shell script for flashing. I ended up renaming it from `make-503.sh` to `make-dhcpd.sh` which seemed more appropriate.